<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ssh_connect_window</class>
 <widget class="QWidget" name="ssh_connect_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>185</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SSH Connection</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/new/prefix2/resources/linux.svg</normaloff>:/new/prefix2/resources/linux.svg</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="server_ip_radio">
       <property name="text">
        <string>Server IP:</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="IP_1">
       <property name="inputMethodHints">
        <set>Qt::ImhDigitsOnly</set>
       </property>
       <property name="inputMask">
        <string>999</string>
       </property>
       <property name="maxLength">
        <number>3</number>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Normal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="dot_1">
       <property name="text">
        <string>.</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="IP_2">
       <property name="inputMethodHints">
        <set>Qt::ImhDigitsOnly</set>
       </property>
       <property name="inputMask">
        <string>999</string>
       </property>
       <property name="maxLength">
        <number>3</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="dot_2">
       <property name="text">
        <string>.</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="IP_3">
       <property name="inputMethodHints">
        <set>Qt::ImhDigitsOnly</set>
       </property>
       <property name="inputMask">
        <string>999</string>
       </property>
       <property name="maxLength">
        <number>3</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="dot_3">
       <property name="text">
        <string>.</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="IP_4">
       <property name="inputMethodHints">
        <set>Qt::ImhDigitsOnly</set>
       </property>
       <property name="inputMask">
        <string>999</string>
       </property>
       <property name="maxLength">
        <number>3</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="port">
       <property name="inputMethodHints">
        <set>Qt::ImhDigitsOnly</set>
       </property>
       <property name="inputMask">
        <string>99999</string>
       </property>
       <property name="maxLength">
        <number>5</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QRadioButton" name="hostname_radio">
       <property name="text">
        <string>Hostname:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="hostname"/>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="hostame_port">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="inputMask">
        <string>99999</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="password_label">
       <property name="text">
        <string>Password:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="password">
       <property name="inputMethodHints">
        <set>Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText|Qt::ImhSensitiveData</set>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="connect">
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
